docs/user/aws/install.md

```diff
@@ -60,5 +63,110 @@ The OpenShift console is available via the kubeadmin login provided by the insta
 
 ![OpenShift web console](images/install_console.png)
 
+## Example: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. This example makes use of and reference to several 
+CloudFormation templates located in this repository.
+
+### Create Ignition Configs
+
+The machines will be manually started. Therefore, it is required to generate the bootstrap and machine ignition configs 
+and store them for later steps.
+
+```console
+[~]$ openshift-install-linux-amd64 create ignition-configs
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform aws
+? Region us-east-1
+? Base Domain openshiftcorp.com
+? Cluster Name test
+? Pull Secret [? for help]
+```
+
+After running the command, several files will be available in the directory.
+
+```console
+[~]$ ls
```

Comment:
nit: Do you want to use `tree` here, so folks can see inside `auth/`?

```console
$ tree
.
├── auth
│   └── kubeconfig
├── bootstrap.ign
├── master.ign
├── metadata.json
└── worker.ign

1 directory, 5 files
```

Reply:
Nice.